etc/alignak.cfg

```diff
@@ -264,8 +264,13 @@ pack_distribution_file=/usr/local/var/lib/alignak/pack_distribution.dat
 
 
 # --------------------------------------------------------------------
-## Arbiter daemon part, similar to daemon ini file
+# Arbiter daemons part, when the arbiter starts some daemons by itself
+# This may happen if some hosts are defined in a realm that do not
+# have all its daemons defined
 # --------------------------------------------------------------------
-#
-# Those parameters are defined in the arbiterd.ini file
-# 
+# Daemons arguments
+#daemons_arguments=
+# Daemons log file
+daemons_log_folder=/usr/local/var/log/alignak
+# Default is to allocate a port number incrementally starting from the value defined here
+daemons_initial_port=7800
```

Comment:
In the code you define per default start at 10000 and in this config you specify 7800. perhaps define the same in the 2 places by default